src/compas_fab/ghpython/path_planning.py

```diff
@@ -8,17 +8,15 @@
 import compas
 from compas.datastructures import Mesh
 from compas.geometry import Transformation
-from compas_ghpython.geometry import xform_from_transformation
 from compas_rhino.geometry import RhinoMesh
+from compas_ghpython.geometry import xform_from_transformation
 
 from compas_fab.robots import Configuration
 
-try:
+if compas.RHINO:
```

Comment:
This doesn't seem to exist in compas 0.15.6.

Reply:
Indeed, it doesn't, but this branch is already set to depend on `>=0.16.1,<0.17`: https://github.com/compas-dev/compas_fab/blob/rename_artists/setup.py#L14

Reply:
oh! strange, usually my ide complains when i don't meet requirements...

Reply:
But it also doesn't exist in 0.16.1 ;)

Reply:
Damn! Semver broken! 🙀  Good catch, I bumped the min patch to be `0.16.2` at https://github.com/compas-dev/compas_fab/pull/201/commits/af671a4bd13a2218367d4879d7f4614b9b8cc4d0